### ISSUE TITLE
`UnnecessaryGetter` issue

### ISSUE THREAD

**Issue by modestukasai** [CodeNarc/CodeNarc#330]
Issue when using spock stubs. If not method is used, test fails...
```
product = Stub(Product) {
                getId() >> 10
            }
```

```
getId() >> 10

[MSG]Violation in class XXX. getId() can probably be rewritten as id
```

**Comment by boris-petrov** [comment]
This is duplicate of https://github.com/CodeNarc/CodeNarc/issues/235.

**Comment by chrismair** [comment]
Fixed under #235. Available in CodeNarc 1.2.
